src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplGetCaseByIdTest.java

```diff
@@ -119,25 +125,97 @@ public void testGetCaseByCaseId_householdIndividualCase() throws CTPException {
     }
   }
 
+  @Test
+  public void testGetLatestFromCacheWhenResultsFromBothRmAndCache() throws Exception {
```

Comment:
I've not looked at the test code yet as I can see that will have quite a few changes following on from the change to the find-case-by-id type changes

Reply:
I have made those changes now and have updated the test code accordingly.